pq_worker.go

```diff
@@ -194,20 +194,9 @@ func (w *PqWorker) getUniqueIdentifier() (Field, error) {
 // that is using this worker
 func (w *PqWorker) consumeRow(row *sql.Row) error {
 	fields := w.Config.Fields
-	s := make([]interface{}, 3)
-	for i, value := range fields {
-		s[i] = value.Pointer
+	var s []interface{}
+	for _, value := range fields {
+		s = append(s, value.Pointer)
 	}
 	return row.Scan(s...)
 }
-
-// consumeNextRow Scans a *sql.Rows into our struct
-// that is using this worker
-func (w *PqWorker) consumeNextRow(rows *sql.Rows) error {
-	fields := w.Config.Fields
-	s := make([]interface{}, 3)
-	for i, value := range fields {
-		s[i] = value.Pointer
-	}
-	return rows.Scan(s...)
-}
```

Comment:
Removing this for now.  This will likely make a comeback if I implement bulk fetch in the future.